### PR TITLE
Correct the package path & License year

### PR DESCRIPTION
**Pull request by chenj47**
Key changes in this pull request:
1. As the new repo is created for Sensor Historian, the path of imported packages are manually changed. Now the code in this new repo can build.
2. Indentation and style are fixed. I ran the golang formatter on all the files, so some indentation and spacing in the files are automatically adjusted.
3. Update the time of the License header for most of the files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,26 +41,26 @@ Next, install Java from <http://www.oracle.com/technetwork/java/javase/downloads
 Next, download the Battery Historian code and its dependencies:
 
 ```
-$ go get -d -u github.com/google/battery-historian/...
+$ go get -d -u github.com/google/sensor-historian/...
```
Comment (agarwal-v) [inline README.md]: URL should be googleinterns/sensor-historian

Comment (agarwal-v) [reply README.md]: This needs to change at multiple places in this file.

File: aggregated/aggregated_stats.go [hunk 2]
```diff
@@ -23,10 +23,10 @@ import (
 	"time"
 
 	"github.com/golang/protobuf/proto"
-	"github.com/google/battery-historian/bugreportutils"
-	"github.com/google/battery-historian/checkinparse"
-	"github.com/google/battery-historian/historianutils"
-	bspb "github.com/google/battery-historian/pb/batterystats_proto"
+	"github.com/google/sensor-historian/bugreportutils"
```
Comment (agarwal-v) [inline aggregated/aggregated_stats.go]: This & various other places - googleinterns

File: pb/build_proto/build.pb.go [hunk 3]
```diff
@@ -13,9 +13,13 @@ It has these top-level messages:
 */
 package build
 
-import proto "github.com/golang/protobuf/proto"
-import fmt "fmt"
-import math "math"
+import (
+	fmt "fmt"
+
+	proto "github.com/golang/protobuf/proto"
```
Comment (agarwal-v) [inline pb/build_proto/build.pb.go:19]: Did you compile pb.go files using protobuf compiler?

**Review by agarwal-v — COMMENTED** [review]
Need to correct github URL to point to googleinterns repo.

File: pb/build_proto/build.pb.go (hunk 3 above)
Comment (chenj47) [reply pb/build_proto/build.pb.go:19]: For some reason, if I compile the pb.go files using protobuf, then the historian tool is not able to run. However, when the code from the old repo, as long as I get the package path correct, everything is working.

**Review by agarwal-v — APPROVED** [review]

**Pull request merged**